cli/export-sqlite-for-user.php

```diff
@@ -5,19 +5,26 @@
 
 performRequirementCheck(FreshRSS_Context::systemConf()->db['type'] ?? '');
 
-$params = [
-	'user:',
-	'filename:',
-];
-
-$options = getopt('', $params);
-
-if (!validateOptions($argv, $params) || empty($options['user']) || empty($options['filename']) || !is_string($options['user']) || !is_string($options['filename'])) {
+$parameters = array(
+	'long' => array(
+		'user' => ':',
+		'filename' => ':',
+	),
+	'short' => array(),
+	'deprecated' => array(),
+);
```

Comment:
Minor syntax detail: Could you please use the `[  ]` array syntax instead of `array( )` for the new code?
(Multiple instance in this file and other modified files)

Reply:
Sure. Just to clarify, shall I replace all instances of `array( )` with `[ ]` in the modified files or just the ones I've introduced?

Reply:
Only your own changes and lines immediately above or below, for now.
We will probably run an automated replacement for the rest in another PR.

Reply:
I think that takes care of it.